_posts/2022-04-13-stepping-out-of-the-knowledge-zone.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: epic
+title: Stepping Out Of The Knowledge Zone
+subtitle: How I went out to face my fears
+date: 2022-04-13
```

Comment:
**suggestion**: Might want to update this to today's date if you're publishing today?